include/Core/Users/Modules/DeleteUsersByUserRoleModule.php

```diff
@@ -38,7 +38,7 @@ protected function initialize() {
 	 * @since 5.5
 	 */
 	public function render() {
-	?>
+		?>
```

Comment:
Looks like some unnecessary whitespace changes got it. In future try to avoid unnecessary white space changes, since they add unnecessary noise to the diff.

Reply:
@sudar It is a phpcs issue, so I fixed it.  In future, I will make sure atomic commits are made.